elasticsearch/config/load_settings.sh

```diff
@@ -15,8 +15,13 @@ else
   wazuh_url="${WAZUH_API_URL}"
 fi
 
+if [ "x${ELASTICSEARCH_USERNAME}" = "x"]; then
```

Comment:
An space need to be added after the "x":
[ "x${ELASTICSEARCH_USERNAME}" = "x" ];
otherwise the following error will occur:
`./load_settings.sh: line 18: [: missing `]'`